CoffeeMachine.java

```diff
@@ -0,0 +1,31 @@
+package oopTwo;
+
+public class CoffeeMachine {
+    Double spentCapacityOfCoffee;
+    Double fullCapacityOfCoffee;
+    Double coffeeCapacityInMachine;
+
+    CoffeeMachine(Double coffeeCapacityInMachine, Double spentCapacityOfCoffee, Double fullCapacityOfCoffee) {
+
+        this.spentCapacityOfCoffee = 60.00;

```

Comment:
What is the purpose of setting this value as 60, and 100 if you want to pass value through the constructor ? 

If you need default values to exist for certain attributes, you can do following:

1. Define initial values for variables:
```
Double spentcapacityOfCoffee = 60.00;
Double fullCapacityOfCoffee = 100.00;
```

2. Create additional constructor which takes only coffeeCapacityInMachine as parameter and sets value, but fix current constructor so that parameter values are passed to the spentCapacityOfCoffee and fullCapacityOfCoffee variables